src/scrappers/amazon-de.yml

```diff
@@ -0,0 +1,24 @@
+url: 'https://www.amazon.de*'
```

Comment:
could we try to use a regex? something like `'^https:\/\/www\.amazon\.(de|es|fr|it|nl).*'`

Reply:
It doesn't work. The extension doesn't even load 
<img width="409" alt="image" src="https://github.com/rows/X/assets/5763905/cff6e35d-2de4-48fb-be9f-ab88d6b2afc7">